OpenDreamRuntime/Objects/Types/DreamObjectArea.cs

```diff
@@ -3,9 +3,11 @@
 public sealed class DreamObjectArea : DreamObjectAtom {
     public int X, Y, Z;
     public readonly AreaContentsList Contents;
+    public int AppearanceId;
 
     public DreamObjectArea(DreamObjectDefinition objectDefinition) : base(objectDefinition) {
         Contents = new(ObjectTree.List.ObjectDefinition, this);
+        AtomManager.SetAtomAppearance(this, new(AtomManager.GetAppearanceFromDefinition(ObjectDefinition)));
```

Comment:
No need to clone the appearance since no modifications will ever be made to it.
```suggestion
        AtomManager.SetAtomAppearance(this, AtomManager.GetAppearanceFromDefinition(ObjectDefinition));
```